### PR TITLE
Remove leftover code in the GeoTIFF parser

### LINKED ISSUE #210: about the GeoTiffReader.js
about the getMetadataFromGeoKeys method of the GeoTiffReader.js file, why didn't you find the value of this.geoKeys? No assignment is given to this.geoKeys.

### PR DESCRIPTION
**Pull request by claudiaifrim**
### Description of the Change
Obsolete method removed from code.

### Why Should This Be In Core?
Obsolete code.

### Benefits
Clean code.

### Potential Drawbacks
None.

### Applicable Issues

Closes #210

### CODE REVIEW

File: src/formats/geotiff/GeoTiffReader.js [hunk 1]
```diff
@@ -888,22 +888,6 @@ define([
             }
         }
 
-        // Get metadata from GeoKeys. Internal use only.
-        GeoTiffReader.prototype.getMetadataFromGeoKeys = function () {
-            for (var i = 0; i < this.geoKeys.length; i++) {
-                var keyAsString = GeoTiffUtil.getTagValueAsString(GeoTiffConstants.Key, this.geoKeys[i].keyId);
```
Comment (AkeluX) [inline src/formats/geotiff/GeoTiffReader.js]: This function indeed looks like obsolete code. In addition, please check whether `GeoTiffUtil.getTagValueAsString` is still needed.

Comment (claudiaifrim) [reply src/formats/geotiff/GeoTiffReader.js]: I reviewed the code and I removed the unused functions:
* getTagValueAsString
* clampColorSampleForElevation
* getMinMaxGeotiffSamples

**Comment by pdavidc** [comment]
@claudiaifrim @AkeluX We're working on simultaneous changes to GeoTiffReader, so I'm reaching out to coordinate. Let's complete and merge this pull request first, as it pertains to removal of obsolete code. Sound good?

**Comment by claudiaifrim** [comment]
@pdavidc , this pull request is ready to be reviewed and merged by @AkeluX

**Pull request merged**